parity-bytes/src/lib.rs

```diff
@@ -19,9 +19,28 @@
 //! Includes a pretty-printer for bytes, in the form of `ToPretty` and `PrettySlice`
 //! as
 
-use std::fmt;
-use std::cmp::min;
-use std::ops::{Deref, DerefMut};
+#![cfg_attr(not(feature = "std"), no_std)]
+#![cfg_attr(not(feature = "std"), feature(alloc))]
```

Comment:
`feature(alloc)` is stable now, no need for this line

Reply:
sure! :ok_hand:

Reply:
`cargo +nightly test --no-default-features` fails,
I think adding `#[macro_use]` should help